clusterloader2/api/types.go

```diff
@@ -20,6 +20,20 @@ import (
 	"time"
 )
 
+// TestSuite defines list of test scenarios to be run.
+type TestSuite []TestScenario
+
+// TestScenario defines customized test to be run.
+type TestScenario struct {
+	// Identifier is a unique test scenario name across test suite.
+	Identifier string `json: identifier`
+	// ConfigPath defines path to the single test config file.
```

Comment:
.. to the file containing a single Config definition.

maybe? or sth like that?

Reply:
Done.